src/client/LiveStream.js

```diff
@@ -515,10 +515,18 @@ exports.StreamStatus = async function({name, stopLro=false, showParams=false}) {
       return status;
     }
 
+    const segDurationMeta = await this.ContentObjectMetadata({
```

Comment:
We already have mainMeta and edgeMeta
We should read seg_duration from edgeMeta (since that is what the stream is currently using)

Reply:
@elv-serban Updated this part